community/javaee8/java11/openj9/Dockerfile

```diff
@@ -62,7 +62,6 @@ RUN mkdir /logs \
     && chown -R 1001:0 /etc/wlp \
     && chmod -R g+rw /etc/wlp \
     && echo "<server description=\"Default Server\"><httpEndpoint id=\"defaultHttpEndpoint\" host=\"*\" /></server>" > /config/configDropins/defaults/open-default-port.xml \
```

Comment:
should we remove the trailing `\` from this line?

Reply:
Yep, it didn't break the build but we probably should.  I'll do that and push the update.